spec/services/quantified_events/create_or_update_service_spec.rb

```diff
@@ -80,6 +80,32 @@
       end
     end
 
+    context 'context without operation type for unique_count_agg' do
```

Comment:
RSpec/ContextWording: Context description should match /^when\b/, /^with\b/, or /^without\b/. (https://rspec.rubystyle.guide/#context-descriptions, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ContextWording)